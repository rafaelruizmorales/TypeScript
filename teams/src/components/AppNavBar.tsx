import * as React from 'react'
import {Link} from 'react-router-dom'
import {Nav, Navbar, NavbarBrand, NavItem, NavLink} from 'reactstrap'

export const AppNavBar: React.FC = () => {
  return (
    <Navbar color="light" light expand="md">
      <NavbarBrand href="/">Football</NavbarBrand>
      <Nav className="mr-auto" navbar>
        <NavItem>
          <NavLink tag={Link} to="/">Players</NavLink>
        </NavItem>
        <NavItem>
          <NavLink tag={Link} to="/teams">Teams</NavLink>
        </NavItem>
        <NavItem>
          <NavLink tag={Link} to="/profile">Profile</NavLink>
        </NavItem>
      </Nav>
    </Navbar>
  )
}
