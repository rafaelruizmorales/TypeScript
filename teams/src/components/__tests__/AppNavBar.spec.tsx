import {shallow} from 'enzyme'
import * as React from 'react'
import {AppNavBar} from '../AppNavBar'

describe('AppNavBar', () => {
  it('should match the snapshot', () => {
    expect(shallow(<AppNavBar />)).toMatchSnapshot()
  })
})
