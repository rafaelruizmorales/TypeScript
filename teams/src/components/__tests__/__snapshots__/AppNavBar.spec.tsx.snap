// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavBar should match the snapshot 1`] = `
<Navbar
  color="light"
  expand="md"
  light={true}
  tag="nav"
>
  <NavbarBrand
    href="/"
    tag="a"
  >
    Football
  </NavbarBrand>
  <Nav
    className="mr-auto"
    navbar={true}
    tag="ul"
    vertical={false}
  >
    <NavItem
      tag="li"
    >
      <NavLink
        tag={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "Link",
            "propTypes": Object {
              "innerRef": [Function],
              "onClick": [Function],
              "replace": [Function],
              "target": [Function],
              "to": [Function],
            },
            "render": [Function],
          }
        }
        to="/"
      >
        Players
      </NavLink>
    </NavItem>
    <NavItem
      tag="li"
    >
      <NavLink
        tag={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "Link",
            "propTypes": Object {
              "innerRef": [Function],
              "onClick": [Function],
              "replace": [Function],
              "target": [Function],
              "to": [Function],
            },
            "render": [Function],
          }
        }
        to="/teams"
      >
        Teams
      </NavLink>
    </NavItem>
    <NavItem
      tag="li"
    >
      <NavLink
        tag={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "Link",
            "propTypes": Object {
              "innerRef": [Function],
              "onClick": [Function],
              "replace": [Function],
              "target": [Function],
              "to": [Function],
            },
            "render": [Function],
          }
        }
        to="/profile"
      >
        Profile
      </NavLink>
    </NavItem>
  </Nav>
</Navbar>
`;
